modules/ROOT/pages/managing-connected-salesforce-orgs.adoc

```diff
@@ -0,0 +1,96 @@
+
+[removing-connection-anypoint-platform]
+=== Remove a Connected Salesforce Org in Anypoint Platform
+
+An Anypoint Platform organization administrator can remove the connection between their Anypoint Platform organization and a connected Salesforce organization.
+
+When you remove a connection to a Salesforce organization, any capabilities that are associated with the Salesforce organization, such as Einstein, are removed. 
+
+To remove the connection to a Salesforce organization:
+
+include::partial$include-nav-salesforce.adoc[]
+
+. Click the *Actions* menu (*...*) in the Salesforce organization to remove the connection from and select *Remove Salesforce Org*. 
+. In *Remove Salesforce Org*, click *Remove Org*.
+
+[removing-connection-salesforce]
+=== Remove a Connected Anypoint Platform Organization in Salesforce
+
+A Salesforce organization administrator can remove the connection between an Anypoint Platform organization and Salesforce organization using the Salesforce user interface. 
+
+To remove the connection to an Anypoint Platform organization in Salesforce:
+
+include::partial$include-salesforce-mulesoft-setup-nav.adoc[]
+
+. In *Your connected Anypoint Platform Organization*, click *View Connection Info*.
+. From *Go To Anypoint*, select *Remove connection*.
+. In *Remove connection to Anypoint Platform*, click *OK*.
+
+Alternatively, if to temporarily prevent Anypoint Platform users from publishing assets to Salesforce from the Anypoint Platform side, you can instead <<enable-disable-connection,disable>> the connection temporarily.
+
+For more information about managing this relationship from the Salesforce user interface, see the https://help.salesforce.com/s/articleView?id=sf.external_services_manage_your_mulesoft_anypoint_platform_connection.htm&type=5[Salesforce documentation]. 
+
+[[repairing-org-connection]]
+== Repair the Connection to a Salesforce Organization
+
+If the Salesforce *My Domain* is changed in the connected Salesforce organization, you can rediscover the Salesforce organization and repair the connection. 
+
+include::partial$include-nav-salesforce.adoc[]
+
+. Click the *Actions* menu (*...*) in the Salesforce org and select *Repair Org Info*.
+. Click *Repair* to rediscover the Salesforce org. 
+
+[[edit-org-name]]
+== Edit the Salesforce Organization Name
+
+You can edit the Salesforce organization display name in your Anypoint Platform organization. The name of the Salesforce organization within Salesforce remains the same. 
+
+To edit a Salesforce organization name:
+
+include::partial$include-nav-salesforce.adoc[]
+
+. Click the *Actions* menu (*...*) in the Salesforce org and select *Edit Org Name*. 
+. Enter the name for the Salesforce org and click *Save*.
++
+The Salesforce organization name appears in Anypoint Platform anywhere the organization is referenced.
+
+== Copy Your Anypoint Platform Organization ID
```

Comment:
Is this used from somewhere else? This task doesn't seem to be a JTBD...

Reply:
This is an independent task.